Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 
 [dependencies]
 serde = { version = "1.0.104", features = ["derive"] }
+serde_yaml = "0.8.11"
```

Comment:
What do you think about following [Specifying Dependencies#Wildcard requirements](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#wildcard-requirements)? 
I would suggest we allow patches in our dependencies (major.minor.*) so that if there are security updates we can get them. Freezing major and minor should keep things behaving in the same way we want.

Reply:
I agree locking only major.minor. There are cases where some crates introduce bugs even with minor upgrade that's why I tend to lock the patch as well. we can try this out however and see how it goes in general.